### PR TITLE
[14.0][IMP] add module to manage ir.rule only in code

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

File: security_rule_not_editable/models/ir_rule.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class IrRule(models.Model):
+    _inherit = "ir.rule"
+
+    def _ensure_install_mode(self):
+        if not self._context.get("install_mode"):
```
Comment (bealdav) [inline security_rule_not_editable/models/ir_rule.py]:
```suggestion
        if not self.env.context.get("install_mode"):
```

File: security_rule_not_editable/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "external_dependencies": {
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "base",
+    ],
+    "data": [],
+    "demo": [],
+    "qweb": [],
```
Comment (bealdav) [inline security_rule_not_editable/__manifest__.py]:
```suggestion
```

**Review by bealdav — APPROVED** [review]
Probably could be useful in oca

**Comment by sebastienbeau** [comment]
@bealdav I am going to battle test it during some month see if there is no issue then I will propose it in OCA

**Pull request merged**